src/main/java/racingcar/RacingCar.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import java.util.Random;
+
+public class RacingCar {
+
+    private static final Random random = new Random();
```

Comment:
정적 상수라면 대문자로 표기해주셔야 합니다 😄 
```suggestion
    private static final Random RANDOM = new Random();
```

Reply:
정적 상수에서 문자열만 주로 대문자로 사용했는데, 정적 상수라면 다 대문자로 써야하군요!
피드백 감사합니다 :)